testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveViewsLegacy.java

```diff
@@ -15,40 +15,35 @@
 
 import io.trino.tempto.BeforeTestWithContext;
 import io.trino.tempto.Requires;
-import io.trino.tempto.assertions.QueryAssert;
 import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableNationTable;
 import io.trino.tempto.fulfillment.table.hive.tpch.ImmutableTpchTablesRequirements.ImmutableOrdersTable;
 import io.trino.tempto.query.QueryExecutor;
 import org.testng.annotations.Test;
 
 import java.math.BigDecimal;
-import java.sql.Date;
-import java.time.LocalDate;
-import java.util.function.Consumer;
 
 import static io.trino.tempto.assertions.QueryAssert.Row.row;
 import static io.trino.tempto.assertions.QueryAssert.assertThat;
 import static io.trino.tempto.query.QueryExecutor.query;
 import static io.trino.tests.TestGroups.HIVE_VIEWS;
-import static io.trino.tests.utils.QueryExecutors.connectToPresto;
 import static io.trino.tests.utils.QueryExecutors.onHive;
 import static io.trino.tests.utils.QueryExecutors.onPresto;
-import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @Requires({
         ImmutableNationTable.class,
         ImmutableOrdersTable.class,
 })
 public class TestHiveViewsLegacy
-        extends HiveProductTest
+        extends AbstractTestHiveViews
 {
     @BeforeTestWithContext
     public void setup()
     {
         setSessionProperty("hive.legacy_hive_view_translation", "true");
     }
 
+    @Override
     @Test(groups = HIVE_VIEWS)
     public void testSelectOnView()
```

Comment:
curious if https://github.com/trinodb/trino/pull/6721#discussion_r566478315 applies here

Reply:
Thanks. The test from super (despite reacher) works fine here too. Removed override.